### PR TITLE
MTM-46693 - Update SSO doc

### PR DESCRIPTION
**Pull request by chbakar**
update SSO screenshots

### CODE REVIEW

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@chbakar Can you please also check the German screenshots and provide new ones if required? Since the User guide is also provided in German we need to have all screenshots with the German UI as well. You can find them in the folder images/benutzerhandbuch (with the same names as the respective English ones).

**Comment by chbakar** [comment]
> @chbakar Can you please also check the German screenshots and provide new ones if required? Since the User guide is also provided in German we need to have all screenshots with the German UI as well. You can find them in the folder images/benutzerhandbuch (with the same names as the respective English ones).

**Comment by chbakar** [comment]
No problem @BeateRixen. I'll update those as well

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 1]
```diff
@@ -277,7 +277,9 @@ Wenn aktiviert, wird die dynamische Rechtezuordnung nur verwendet, wenn sich ein
 
 Wenn der Benutzer sich mit einem Access Token anmeldet, kann der Benutzername aus einem JWT-Claim abgeleitet werden. Der Name des Claims kann unter **Benutzer-ID** konfiguriert werden.
 
- ![OAuth configuration](/images/benutzerhandbuch/Administration/admin-sso-3.png)
+Die Benutzer-ID kann auf ein beliebiges Feld des an die Plattform gesendeten Autorisierungs-Tokens gesetzt werden. Wir empfehlen Ihnen, das Autorisierungs-Token in den Audit-Logs zu überprüfen, um sicherzustellen, dass das richtige Feld verwendet wird (siehe [Troubleshooting](#troubleshooting)).
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:280]:
```suggestion
Die Benutzer-ID kann in ein beliebiges Feld des an die Plattform gesendeten Autorisierungs-Tokens gesetzt werden. Wir empfehlen Ihnen, das Autorisierungs-Token in den Audit-Logs zu überprüfen, um sicherzustellen, dass das richtige Feld verwendet wird (siehe [Troubleshooting](#troubleshooting)).
```

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 2]
```diff
@@ -303,7 +305,7 @@ Jedes Access Token wird durch ein Signing-Zertifikat signiert. Aktuell gibt es d
 
  ![OAuth configuration](/images/benutzerhandbuch/Administration/admin-sso-5.png)
 
-4. Durch Spezifizieren der JWKS (JSON Web Key Set)-Adresse.
+4. Durch Spezifizieren der JWKS (JSON Web Key Set)-Adresse. JWKS ist ein Satz kryptografischer Schlüssel, die öffentliche Schlüssel enthalten, die zum Verifizieren von Autorisierungs-Token verwendet werden, die vom Autorisierungsserver ausgestellt wurden.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:308]:
```suggestion
4. Durch Spezifizieren der JWKS (JSON Web Key Set)-Adresse. JWKS ist ein Satz kryptografischer Schlüssel, die öffentliche Schlüssel enthalten, die zum Verifizieren von Autorisierungs-Tokens verwendet werden, die vom Autorisierungsserver ausgestellt wurden.
```

**Review by BeateRixen — APPROVED** [review]

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 3]
```diff
@@ -277,7 +277,9 @@ Wenn aktiviert, wird die dynamische Rechtezuordnung nur verwendet, wenn sich ein
 
 Wenn der Benutzer sich mit einem Access Token anmeldet, kann der Benutzername aus einem JWT-Claim abgeleitet werden. Der Name des Claims kann unter **Benutzer-ID** konfiguriert werden.
 
- ![OAuth configuration](/images/benutzerhandbuch/Administration/admin-sso-3.png)
+Die Benutzer-ID kann in ein beliebiges Feld des an die Plattform gesendeten Autorisierungs-Tokens gesetzt werden. Wir empfehlen Ihnen, das Autorisierungs-Token in den Audit-Logs zu überprüfen, um sicherzustellen, dass das richtige Feld verwendet wird (siehe [Troubleshooting](#troubleshooting)).
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:280]: @MWindel FYI. This is the German file which will contain new text.

File: content/users-guide/administration-bundle/changing-settings.md [hunk 4]
```diff
@@ -358,7 +357,7 @@ Each access token is signed by a signing certificate. Currently there are three
 
  ![OAuth configuration](/images/users-guide/Administration/admin-sso-5.png)
 
-4. By specifying the JWKS (JSON Web Key Set) address.
+4. By specifying the JWKS (JSON Web Key Set) address. JWKS is a set of cryptographic keys containing public keys used to verify tokens issued by the authorization server.
```
Comment (micst96) [inline content/users-guide/administration-bundle/changing-settings.md:360]:
```suggestion
4. By specifying the JWKS (JSON Web Key Set) URI. JWKS is a set of JWK objects containing a public key used to verify tokens issued by the authorization server.
```

**Review by micst96 — CHANGES_REQUESTED** [review]

File: content/users-guide/administration-bundle/changing-settings.md [hunk 5]
```diff
@@ -331,8 +329,9 @@ By default, dynamic access mapping assigns user roles, based on the access token
 When selected, dynamic access mapping will be used only when a new user logs in to fill in the initial roles. When a user already exists in {{< product-c8y-iot >}}, the roles will not be overridden nor updated. Selecting this option also enables admins to edit roles of SSO users in the user management. For details, refer to  [Administration > Managing permissions](/users-guide/administration/#attach-global) in the *User guide*.
 
 When a user logs in with an access token, the username can be derived from a JWT claim. The claim name can be configured in the **User ID configuration** window.
-
- ![OAuth configuration](/images/users-guide/Administration/admin-sso-3.png)
+The user ID can be set to any field of the authorization token sent to the platform. We recommend you to inspect the authorization token in the audit logs to make sure the correct field is used (see [Troubleshooting](#troubleshooting)).
```
[the remark this answers is not in the document]
Comment (chbakar) [reply content/users-guide/administration-bundle/changing-settings.md:332]: The top-level fields is definitely more detailed

**Review by micst96 — APPROVED** [review]

**Pull request merged**